packages/rspack-plugin-react-refresh/src/options.json

```diff
@@ -0,0 +1,33 @@
+{
+  "additionalProperties": false,
```

Comment:
Let's use zod

Reply:
I copied it from react-fast-refresh plugin.

Reply:
it's slow, and let's avoid bring another schema validation lib into our codebase, and schema validate at here seems not necessary, so I prefer remove it or use zod

Reply:
To be honest, the loader-runner also has`schema-utils` builtin to validate loader options. So I believe it's already there.

> schema validate at here seems not necessary

Any reason for that?

Reply:
emm, seems we can't avoid using schema-utils..

Reply:
I was thinking of converting json-schema to zod in order to remove schema-utils completely.

Reply:
That would be great, but I'm not sure whether there is a library in community to converting json-schema to zod at runtime, and including the convert time cost it can still faster than schema-utils